02.recognize_digits/README.md

```diff
@@ -245,6 +245,15 @@ def train_program():
     return [avg_cost, acc]
 ```
 
+#### Optimizer Function Configuration
+
+In the following `Adam` optimizer, `learning_rate` means the speed at which the network training converges.
```

Comment:
Can we rephrase: `learning_rate` specifies the learning rate in the optimization procedure.

Reply:
sure, sounds good.